src/backend/base/langflow/components/outputs/chat.py

```diff
@@ -184,10 +187,7 @@ def _safe_convert(self, data: Any) -> str:
             if isinstance(data, Message):
                 return data.get_text()
             if isinstance(data, Data):
-                if data.get_text() is None:
-                    msg = "Empty Data object"
-                    raise ValueError(msg)
-                return data.get_text()
+                return "```json\n" + json.dumps(data.data, indent=4) + "\n```"
```

Comment:
You should use jsonable_encoder on the data because .data may contain non-serializable objects.

Reply:
And probably use orjson